### PR TITLE
Add french FNSO funder

### PR DESCRIPTION
**Pull request by yvanlebras**

### CODE REVIEW

File: FUNDERS.yaml [hunk 1]
```diff
@@ -238,3 +238,17 @@ eurosciencegateway:
...
+    avatar: "https://gallantries.github.io/assets/images/fnso.png"
+    github: false
+    funder: true
+    url: "https://www.ouvrirlascience.fr/national-fund-for-open-science/"
+    funder_name: National Fund for Open Science
+    funding_id: AAPFNSO2019OpenMetaPaper-14026
+    funding_statement: |
+        This project ([`AAPFNSO2019OpenMetaPaper-14026`](https://www.ouvrirlascience.fr/submit-your-data-its-published/)) is funded with the support of the national Fund for Open Science programme of the French Minister of Higher Education, Research and Innovation. Their funding has supported development of data/metadata management tools and related tutorial for Ecology and FAIR topics.
+        ![French research ministry flag with the text: with the support of the National Fund for Open Science programme of french Minister of Higher Education, Research and Innovation](https://lamy-environnement.com/wp-content/uploads/2023/03/Logo-MESR.png)
+        ![FNSO logo](https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR2KP3c0FszeNgfGvcc-0ipd-Tt6DE3VFF7TeQeX5PO-A&s)
```
Comment (hexylena) [inline FUNDERS.yaml:254]:
```suggestion
        ![FNSO logo]({% link shared/images/fsno.png %})
```

File: FUNDERS.yaml [hunk 2]
```diff
@@ -238,3 +238,17 @@ eurosciencegateway:
...
+    joined: 2023-05
+    avatar: "https://gallantries.github.io/assets/images/fnso.png"
+    github: false
+    funder: true
+    url: "https://www.ouvrirlascience.fr/national-fund-for-open-science/"
+    funder_name: National Fund for Open Science
+    funding_id: AAPFNSO2019OpenMetaPaper-14026
+    funding_statement: |
+        This project ([`AAPFNSO2019OpenMetaPaper-14026`](https://www.ouvrirlascience.fr/submit-your-data-its-published/)) is funded with the support of the national Fund for Open Science programme of the French Minister of Higher Education, Research and Innovation. Their funding has supported development of data/metadata management tools and related tutorial for Ecology and FAIR topics.
+        ![French research ministry flag with the text: with the support of the National Fund for Open Science programme of french Minister of Higher Education, Research and Innovation](https://lamy-environnement.com/wp-content/uploads/2023/03/Logo-MESR.png)
```
Comment (hexylena) [inline FUNDERS.yaml:253]: the alt text :heart_eyes: :heart_eyes: merci !

Comment (yvanlebras) [reply FUNDERS.yaml:253]: not sure if you are saying it is very good or bad ? ;) sorry

Comment (hexylena) [reply FUNDERS.yaml:253]: yes it's good! and always so nice when people write it and I do not have to nag them :)

**Review by hexylena — APPROVED** [review]

**Pull request merged**